### PR TITLE
Update social embed translations

### LINKED ISSUE #6123: Social embeds - add latest translations
**Is your feature request related to a problem? Please describe.**
Some translations for social embeds are missing, as they weren't ready in time for initial work (see https://github.com/bbc/simorgh/issues/6005).

**Describe the solution you'd like**
Add missing translations to the [service config files](https://github.com/bbc/simorgh/pull/6024/files). Translations can be found in the translations spreadsheet.

Also fix this issue that was found during the social embed a11y swarm: 
Mundo: Figure caption for YouTube should have a comma afterwards, not a colon, a comma creates a pause with AT.

**Testing notes**

- [ ] This feature is expected to need manual testing.

**Additional context**

Details of what is missing can be found in NEWSART-917.

### PR DESCRIPTION
**Pull request by chewu**
Resolves https://github.com/bbc/simorgh/issues/6123

**Overall change:**
Add missing translations to the service config files. Please note, some translations are still in English.

**Code changes:**
Update translations to socialEmbed: config

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by rhenshaw56 — APPROVED** [review]

File: src/app/lib/config/services/mundo.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ export const service = {
       },
       socialEmbed: {
         caption: {
-          textPrefixVisuallyHidden: 'Título del video: ',
+          textPrefixVisuallyHidden: 'Título del video, ',
```
Comment (simonsinclair) [inline src/app/lib/config/services/mundo.js:174]: Good catch!

**Review by chris-hinds — APPROVED** [review]
Jenkins failed on the JS bundle tests, nothing todo with this PR. I have re-run the pipeline

File: src/app/lib/config/services/serbian.js [hunk 2]
```diff
@@ -259,18 +259,18 @@ export const service = {
...
+          text: 'Sadržaj nije dostupan',
+          linkText: 'Pogledajte više na %provider_name%',
+          linkTextSuffixVisuallyHidden: ', spoljna stranica',
           warningText: 'BBC nije odgovoran za sadržaj drugih sajtova.',
         },
         skipLink: {
-          text: 'Skip %provider_name% post',
-          endTextVisuallyHidden: 'End of %provider_name% post',
+          text: 'Preskočite sadržaj sa %provider_name%',
+          endTextVisuallyHidden: 'Kraj sadržaja sa [%provider_name%',
```
Comment (simonsinclair) [inline src/app/lib/config/services/serbian.js]: Is the opening square bracket `[` intentional?

**Review by simonsinclair — COMMENTED** [review]
Thanks for tackling this, Che. 👍

File: src/app/lib/config/services/serbian.js (hunk 2 above)
Comment (chewu) [reply src/app/lib/config/services/serbian.js]: @simonsinclair Let me fix this 👍

**Comment by paruchurisilpa** [comment]
This doesn't require any manual testing, moving it to Ready to merge..

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**